2-WebApp-graph-user/2-2-TokenCache/README.md

```diff
@@ -26,7 +26,7 @@ It leverages the ASP.NET Core OpenID Connect middleware and Microsoft Authentica
 
 To run this sample, you'll need:
 
-- [Visual Studio 2017](https://aka.ms/vsdownload) or just the [.NET Core SDK](https://www.microsoft.com/net/learn/get-started)
+- [Visual Studio 2019](https://aka.ms/vsdownload) or just the [.NET Core SDK](https://www.microsoft.com/net/learn/get-started)
```

Comment:
I wonder if we want to keep giving version :)

Reply:
no I don't think this makes sense, I'll remove it

Reply:
fixed